src/pybricks/parameters.py

```diff
@@ -36,7 +36,7 @@
 
 
 class _PybricksEnumMeta(type(Enum)):
-    def __dir__(cls):
+    def __dir__(cls):   # noqa: N805
```

Comment:
If this is a class method, shouldn't we just decorate it with `@classmethod`?

Reply:
@dlech I thought so too but was not confident of its full implication to the lib. Shall I just decorate with `@classmethod` and then leave end-to-end testing to you?

Reply:
Yes, that sounds good.